lib/benchee/conversion/count.ex

```diff
@@ -76,7 +84,7 @@ defmodule Benchee.Conversion.Count do
   ## Examples
 
       iex> Benchee.Conversion.Count.scale(12345, :one)
-      12345
+      12345.0
```

Comment:
Yeah had the same thought, could get around it with a pattern match but I even sort of liked it, as then the return value is a float more... uniformly :)